addons/account/models/account_move.py

```diff
@@ -374,6 +374,10 @@ def _get_default_invoice_incoterm(self):
     # Technical field to hide Reconciled Entries stat button
     has_reconciled_entries = fields.Boolean(compute="_compute_has_reconciled_entries")
     show_reset_to_draft_button = fields.Boolean(compute='_compute_show_reset_to_draft_button')
+    # Credit Limit related fields
+    partner_credit = fields.Monetary(related='partner_id.credit', readonly=True)
+    partner_credit_limit = fields.Monetary(related='partner_id.credit_limit', readonly=True)
+    partner_credit_warning = fields.Boolean(compute='_compute_partner_credit_warning')
```

Comment:
As it's a boolean, it would be nice to have it starting by "is"
`is_partner_limit_reached` would be nice for what it represents.

`is_credit_limit_warning_displayed` is also a possibility (but then it will let people think it's for display only until we change it.)